### PR TITLE
Add check spec for network list

### PR DESCRIPTION
**Pull request by irisZhangCB**
Fixes # .

### Motivation

Add check spec for network list

### CODE REVIEW

File: cmd/check_spec.go [hunk 1]
```diff
@@ -110,6 +138,23 @@ func (cs *checkSpec) NetworkStatus(ctx context.Context) error {
...
+func (cs *checkSpec) NetworkList(ctx context.Context, fetcher *fetcher.Fetcher) error {
+	networks, err := fetcher.NetworkList(ctx, nil)
+	if err != nil {
+		return fmt.Errorf("%w: unable to fetch network list", err.Err)
+	}
+	if len(networks.NetworkIdentifiers) == 0 {
+		return fmt.Errorf("network_identifiers are required")
+	}
+	for _, network := range networks.NetworkIdentifiers {
+		if types.Hash(network.Network) == types.Hash(Config.Network.Network) &&
```
[the remark this answers is not in the document]
Comment (shiatcb) [reply cmd/check_spec.go:150]: And also in line 151: network.Blockchain == Config.Network.Blockchain?

[the remark this answers is not in the document]
Comment (shiatcb) [reply cmd/check_spec.go]: Oh I c, both network.Network and Config.Network.Network are pointers, that might be the reason to use types.Hash. Would you mind running the test locally and verify if types.Hash is needed?

[the remark this answers is not in the document]
Comment (shiatcb) [reply cmd/check_spec.go]: Confused myself. Both are strings, not pointers. In this case I don't think we need types.Hash.

**Review by shiatcb — APPROVED** [review]

**Comment by cb-heimdall** [comment]
Review Error for shiatcb @ 2022-06-09 17:06:06 UTC
User must have write permissions to review

**Review by shrimalmadhur — APPROVED** [review]

**Pull request merged**